lib/stdlib/doc/src/zip.xml

```diff
@@ -506,6 +506,14 @@
       </desc>
     </func>
 
+    <func>
+      <name name="zip_get_crc32" arity="2" since="OTP 25.x"/>
```

Comment:
For this PR to appear in OTP 25.1, you'll need to rebase it on the `maint` branch and edit the PR to change its target to `maint`.

To get the correct version number, do the following change and our scripts will ensure that the correct version is inserted at the time of the release:

```suggestion
      <name name="zip_get_crc32" arity="2" since="OTP @OTP-18159"/>
```

Reply:
Cool, I've merged that. There is no need to be in 25.1, can be a later version as well. Is @OTP-18159 a magic number that will always be replaced?

Reply:
`@OTP-18159@` is a magic token that will only work for this pull request.

I now see that I forgot the `@` at the end. That is, the line should be:

```
      <name name="zip_get_crc32" arity="2" since="OTP @OTP-18159@"/>
```

> There is no need to be in 25.1, can be a later version as well.

The next later version for the `master` branch will be OTP 26.0 in about  a years time.